vegaprotocol/vega
# Support hot-reload of the long-living tokens

Close #7093 

## Test protocol

1. start the service: `vega wallet service run -n fairground --no-version-check --load-tokens`
2. generate a token: `vega wallet api-token genererate ...`
3. try to call `client.list_keys` using the generated token. It should return the key list.
4. delete this token: `vega wallet api-token delete`
5. try to call `client.list_keys` using the generated token. It should return a descriptive error message.
6. generate a token, _again_: `vega wallet api-token genererate ...`
7. try to call, _again_ `client.list_keys` using the generated token. It should return the key list.
8. Delete the token file entirely,
9. try to call `client.list_keys` using the generated token. It should return a descriptive error message.
